pkg/rancher-components/src/components/Form/Checkbox/Checkbox.test.ts

```diff
@@ -0,0 +1,77 @@
+import { shallowMount } from '@vue/test-utils';
+import { Checkbox } from './index';
+
+describe('Checkbox.vue', () => {
+  it('renders falsy by default', () => {
```

Comment:
This is really a nitpick, but I think a name like `it('is unchecked by default')` might be a little easier to read.

tbh I think I've named a test or two like this recently 🤐

Reply:
I may have acquired your naming schemes a bit  :disguised_face: